.github/workflows/ci.yml

```diff
@@ -32,20 +32,11 @@ jobs:
       fail-fast: false
       matrix:
         ruby-version:
-          - 2.4
-          - 2.5
           - 2.6
```

Comment:
Shall we add a `required_ruby_version` in the gemspec?

Reply:
Good suggestion, since we're not testing with that version then I would say yes, we should set `required_ruby_version` to 2.6